src/components/SettingsPage/CounterValueExchangeSelect.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import React, { Fragment, PureComponent } from 'react'
+import { connect } from 'react-redux'
+import { createStructuredSelector } from 'reselect'
+import {
+  counterValueCurrencySelector,
+  counterValueExchangeSelector,
+  intermediaryCurrency,
+} from 'reducers/settings'
+import { setCounterValueExchange } from 'actions/settings'
+import type { Currency } from '@ledgerhq/live-common/lib/types'
+import SelectExchange from 'components/SelectExchange'
+
+type Props = {
+  counterValueCurrency: Currency,
+  counterValueExchange: ?string,
+  setCounterValueExchange: (?string) => void,
+}
+
+class CounterValueExchangeSelect extends PureComponent<Props> {
+  handleChangeExchange = (exchange: *) =>
+    this.props.setCounterValueExchange(exchange ? exchange.id : null)
+
+  render() {
+    const { counterValueCurrency, counterValueExchange } = this.props
+
+    return (
+      <Fragment>
+        {counterValueCurrency ? (
+          <SelectExchange
+            small
+            from={intermediaryCurrency}
+            to={counterValueCurrency}
+            exchangeId={counterValueExchange}
+            onChange={this.handleChangeExchange}
+            minWidth={200}
+          />
+        ) : null}
+      </Fragment>
```

Comment:
I think you can get rid of Fragment here. Also, I think it would be better to always render a select (in loading state for example, (to be coded)) instead of null, to prevent any blink

Reply:
you're right. i know. i know. but we'll keep them because very soon. like in a few days. we'll have to actually `<Track/>` this, so we'll need it :P

agree for the blink tho ;) i'll fix

Reply:
Ooh ok got it :ok_hand: